.env.development

```
@@ -5,4 +5,5 @@ VITE_APP_TITLE = 管理系统模板
 VITE_APP_ENV = 'development'
 
 # 管理系统模板/开发环境
-VITE_APP_BASE_URL = 'http://localhost:80'
+# VITE_APP_BASE_URL = 'http://localhost:80'
+VITE_APP_BASE_URL = 'http://192.168.110.68:6088'
```

dotenv-linter (3.3.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)